### DISCUSSION TITLE
Inability to choose refreshrate: 23.976hz.

### DISCUSSION THREAD

**Discussion by 6nightrhyme6** [Codectory/AutoActions#139] [General]
Hi

Really enjoying your app. But I really need to be able to choose 23,976hz to match framerate of most 4k HDR movies
Would it be possible to add this feature

Thanx in advance

**Comment by Fr0stX76** [comment]
Try 23hz. On Nvidia driver, 23 is actually 23.976

**Comment by 6nightrhyme6** [comment]
Thanx for reply mate

Yes that worked like a charm. Confirmed in adapter properties that it is actually running at 23,976hz.
For reference I'm not running NVIDIA but an Intel Nuc12 mini pc with Intel® Iris® Xe Graphics.
